tests/pinkopy/test_exceptions.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+
+import pytest
+import requests
+
+from pinkopy import exceptions
+from pinkopy.exceptions import PinkopyError
+from tests.pinkopy import test_helper
+
+
+class TestPinkopyError(unittest.TestCase):
+    msg = test_helper.get_uuid()
+    try:
+        raise PinkopyError(msg)
+    except PinkopyError as e:
+        assert msg == e.args[0]
+
+
+class TestModuleMethods(unittest.TestCase):
+    def test_raise_requests_error(self):
+        msg = test_helper.get_uuid()
+        status_code = test_helper.get_uuid()
+        try:
+            with pytest.raises(requests.HTTPError):
+                exceptions.raise_requests_error(status_code, msg)
+        except requests.HTTPError as e:
+            assert msg == e.args[0]
```

Comment:
Testing that the helper `raise_requests_error` actually does so. I dig it. Might as well test that the status code got through, too.

```python
status_code, mss = 404, 'NOT FOUND'
try:
    with pytest.raises(requests.HTTPError):
        exceptions.raise_requests_error(status_code, msg)
except requests.HTTPError as err:
    assert err.args[0] == msg
    assert err.response.status_code == status_code
```